src/main/webapp/scripts/keywords.js

```diff
@@ -15,3 +15,34 @@ $().ready(function() {
     $('#form-title *').text('Copy Your Tweet Here');
   });
 });
+
+/** 
+ * Checks whether the form inputs are valid before submitting said form.
+ */
+function validateForm(event) {
+  if ($('#blob-input').hasClass('hide')) {
+    // Check the text input
+    if ($('#text-input').val() == null || $('#text-input').val() === "") {
+      alert('Your input cannot be empty!'); 
+      event.preventDefault();
+      return false;
+    } else {}
```

Comment:
[Nit] Remove empty else

Reply:
This is needed so that the next else statement does not get attributed to the wrong if statement.

Reply:
Just kidding, this is not needed.